src/templates/static/metatrader/download.jsx

```diff
@@ -1,108 +1,228 @@
-                    download='true'
-                />
-                <DownloadBlock
-                    title={it.L('Linux')}
-                    desc={it.L('All versions of Linux')}
-                    image='linux'
-                    href='https://www.metatrader5.com/en/terminal/help/start_advanced/install_linux'
-                    target='_blank'
-                />
-            </div>
-            <p className='foot-note gr-padding-20'>{it.L('The MetaTrader platform will no longer support Windows XP, Windows 2003, and Windows Vista after 01 October, 2017.') }</p>
+            <DownloadButton
+                badge={badge}
+                href={href}
+                download={download}
+                os={os}
+                target={target}
+            >
+                {it.L(`Download MT5 for ${title}`)}
+            </DownloadButton>
         </div>
+    </div>
+);
 
-        <SeparatorLine no_wrapper />
+const Download = () => {
+    const windows_link = `<a data-type='alt-link' data-target='windows' id='enable-windows-alt-link'>${it.L('Windows')}</a>`;
+    const linux_link = `<a data-type='alt-link' data-target='linux' id='enable-linux-alt-link'>${it.L('Linux')}</a>`;;
+    const mac_link = `<a data-type='alt-link' data-target='mac' id='enable-mac-alt-link'>${it.L('Mac')}</a>`;;
+    return (
+        <div id='mt_download' className='static_full center-text'>
+            <h1>{it.L('Start Trading with MetaTrader 5')}</h1>
+            <h3>{it.L('Trade with a powerful interface known as the global industry standard.')}</h3>
 
-        <div className='gr-padding-20'>
-            <h3>{it.L('For Mobile') }</h3>
-            <p>{it.L('Access the markets anytime, anywhere using native apps for your iOS or Android devices.') }</p>
-            <div className='gr-row'>
-                <div className='gr-2 gr-hide-m' />
-                <DownloadBlock
-                    image='android'
-                    os='android'
-                    title={it.L('Android')}
-                    desc={it.L('All versions of Android')}
-                    target='_blank'
-                    badge='google-play-badge'
-                />
-                <DownloadBlock
-                    image='apple'
-                    os='ios'
-                    title={it.L('iOS')}
-                    desc={it.L('All versions of iOS')}
-                    target='_blank'
-                    badge='app-store-badge'
-                />
+            <div className='gr-padding-20'>
+                <h1 className='invisible download-heading' id='windows-heading'>
+                    {it.L('MT5 for Windows')}
+                </h1>
+                <h1 className='invisible download-heading' id='macos-heading'>
+                    {it.L('MT5 for Mac')}
+                </h1>
+                <h1 className='invisible download-heading' id='linux-heading'>
+                    {it.L('MT5 for Linux')}
+                </h1>
```

Comment:
these h1 elements also look like they could be turned into a block and reused. if you change macos to mac, you can only pass in one parameter to use both for the id and the header text, just by making it lowercase in id